src/neural/cuda/network_cudnn.cc

```diff
@@ -272,12 +275,37 @@ class CudnnNetwork : public Network {
     // support it).
     ReportCUBLASErrors(cublasSetMathMode(cublas_, CUBLAS_TENSOR_OP_MATH));
 
-    // Check if we want to enable our custom winograd ?
     constexpr bool fp16 = std::is_same<half, DataType>::value;
+    const int kNumInputPlanes = kInputPlanes;
+    const int kNumFilters = weights.input.biases.size();
+    numBlocks_ = weights.residual.size();
+
+    // Use our custom winograd for resudual tower convolutions for most cases:
```

Comment:
typo `resudual`

Reply:
fixed. Really need to be more careful :-)